alert_group_nl_log2slack.py

```diff
@@ -181,18 +185,21 @@ def dump_cookies(session, where):
 def login_and_fetch(klant_nummer, klant_gecrypt):
     # Session keeps cookies around.
     with requests.Session() as session:
-        ret = session.get(ALERTMOBILE_URL)
+        ret = session.get(ALERTMOBILE_URL,
+                          timeout=10)
         dump_cookies(session, 'first get')
         assert ret.status_code == 200, (ret, ret.text)
 
         ret = session.post(ALERTMOBILE_URL, data={
                 'klantnr': klant_nummer, 'klantcode': '',
-                'gecrypt': klant_gecrypt})
+                'gecrypt': klant_gecrypt},
+                timeout=10)
```

Comment:
See above.

Reply:
I'm assuming you're referring to the "no need for this to be on 2 lines", so I'll fix that.